webpack/webpack.config.base.js

```diff
@@ -19,7 +19,7 @@ module.exports = function getBaseConfig(config) {
     context: process.cwd(),
     output: {
       path: config.assetsPath,
-      publicPath: `/${relativeAssetsPath}/`,
+      publicPath: process.env.PUBLIC_PATH || `/${relativeAssetsPath}/`,
```

Comment:
Reading directly from a process environment variable doesn't make sense for an open source project, The relativeAssetPath should be an option in the config object, that can be altered from the delivery